src/Plugin.php

```diff
@@ -82,94 +75,22 @@ public function uninstall(Composer $composer, IOInterface $io): void
-        $datastore = self::buildDefaultDatastore($config, $log);
-        $classMapGenerator = new MemoizeClassMapGenerator($datastore, $log);
-        foreach ($config->include as $include) {
-            $classMapGenerator->scanPaths($include, $config->excludeRegExp);
-        }
-        $classMap = $classMapGenerator->getMap();
-        $elapsed = self::renderElapsedTime($start);
-        $log->debug("Generating attributes file: scanned paths in $elapsed");
-
-        //
-        // Filter the class map
-        //
-        $start = microtime(true);
-        $classMapFilter = new MemoizeClassMapFilter($datastore, $log);
-        $filter = self::buildFileFilter();
-        $classMap = $classMapFilter->filter(
-            $classMap,
-            fn (string $class, string $filepath): bool => $filter->filter($filepath, $class, $log)
-        );
-        $elapsed = self::renderElapsedTime($start);
-        $log->debug("Generating attributes file: filtered class map in $elapsed");
-
-        //
-        // Collect attributes
-        //
         $start = microtime(true);
-        $attributeCollector = new MemoizeAttributeCollector(new ClassAttributeCollector($log), $datastore, $log);
-        $collection = $attributeCollector->collectAttributes($classMap);
-        $elapsed = self::renderElapsedTime($start);
-        $log->debug("Generating attributes file: collected attributes in $elapsed");
-
-        //
-        // Render attributes
-        //
-        $start = microtime(true);
-        $code = self::render($collection);
-        file_put_contents($config->attributesFile, $code);
-        $elapsed = self::renderElapsedTime($start);
-        $log->debug("Generating attributes file: rendered code in $elapsed");
-    }
-
-    private static function buildDefaultDatastore(Config $config, Logger $log): Datastore
-    {
-        if (!$config->useCache) {
-            return new RuntimeDatastore();
-        }
-
-        $basePath = Platform::getCwd();
-
-        assert($basePath !== '');
-
-        return new FileDatastore($basePath . DIRECTORY_SEPARATOR . self::CACHE_DIR, $log);
-    }
-
-    private static function renderElapsedTime(float $start): string
-    {
-        return sprintf("%.03f ms", (microtime(true) - $start) * 1000);
-    }
-
-    private static function buildFileFilter(): Filter
-    {
-        return new Filter\Chain([
-            new ContentFilter(),
-            new InterfaceFilter()
-        ]);
+        self::dump($config);
+        $elapsed = ElapsedTime::render($start);
+        $io->write("<info>Generated attributes file in $elapsed</info>");
     }
 
-    private static function render(TransientCollection $collector): string
+    public static function dump(Config $config): void
     {
-        return TransientCollectionRenderer::render($collector);
+        $cmd = __DIR__ . '/../collector.php';
+        $serializedConfig = escapeshellarg(serialize($config));
+        system("$cmd -- $serializedConfig");
```

Comment:
I think you should use Symfony Process here, which should already be available in Composer context. See how Composer itself runs commands such as "git" and so on.